thunder/tests/test_update_aliases.py

```diff
@@ -3,8 +3,10 @@
 import pytest
 import torch.testing
 
+import thunder
```

Comment:
ditto, it doesn't seem that we use this import
```suggestion
```

Reply:
This import is used in `test_inplace_to_alias_func_args`.